src/Microsoft.Azure.WebJobs.Host/FunctionExecutingContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// The context for an executed function
+    /// </summary>
+    [CLSCompliant(false)]
```

Comment:
Ah, I see, thanks. We might separately investigate just removing our CLS compliance, since this is just a headache that doesn't really add value, and forces us to sprinkle these attributes all over everywhere.